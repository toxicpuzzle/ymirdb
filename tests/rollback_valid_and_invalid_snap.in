SET a 1
SNAPSHOT  
SET b 2
SNAPSHOT
SET c 3
SNAPSHOT
LIST SNAPSHOTS
LIST ENTRIES
ROLLBACK 1
ROLLBACK 2
ROLLBACK 0
LIST ENTRIES
LIST SNAPSHOTS
DROP 1
LIST ENTRIES
bye