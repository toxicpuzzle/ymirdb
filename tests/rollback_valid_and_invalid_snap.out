> ok

> saved as snapshot 1

> ok

> saved as snapshot 2

> ok

> saved as snapshot 3

> 3
2
1

> c [3]
b [2]
a [1]

> ok

> no such snapshot

> no such snapshot

> a [1]

> 1

> ok

> a [1]

> bye
